Scripts/script-Cut.yml

```diff
@@ -0,0 +1,36 @@
+commonfields:
+  id: cae96081-c034-46c1-81f6-b4d0a52dc51c
+  version: 38
+name: Cut
+script: |-
+  args = demisto.args()
+
+  value = args.get('value')
+  fields = args.get('fields')
+  delim = args.get('delimiter')
+
+  data = value.split(delim)
+  fields = [int(_) for _ in fields.split(',')]
+
+  demisto.results(delim.join([str(data[i-1]) for i in fields]))
+type: python
+tags:
+- transformer
+- string
+comment: Cut a string by delimiter and return specific fields
```

Comment:
lets consult with Alex about this comment.

I would give an example (e.g `Cut value=<val> delimiter=<del> ... => <result-example>`)